README.md

```diff
@@ -1,18 +1,49 @@
 # Faros Events CLI
 CLI for reporting events to Faros platform
 
-## Example Emits
+## `./faros_event.sh` Usage Examples
+
+### Sending a Build Event
+
+```sh
+./faros_event.sh build -k "<api_key>" \
+    --application "<app_name>" \
+    --build_status "<build_status>" \
+    --ci_org "<ci_organization>" \
+    --commit "<commit_sha>" \
+    --repo "<vcs_repo>" \
+    --pipeline "<ci_pipeline>" \
+    --vcs_source "<vcs_source>" \
+    --vcs_org "<vcs_organization>" \
+    --print_event
+```
+
+### Sending a Deployment Event
 
 ```sh
-# You need not replace any
-./faros_emit.sh full -k "<api_key>" -a "<app_name>" -e "<environmnet>" -c "<commit_sha>" --build_status "<build_status>" --deploy_status "<deploy_status>" --repo "<vcs_repo>" -p "<ci_pipeline>" --ci_org "<ci_organization>" --vcs_source "<vcs_source>" --vcs_org "<vcs_organization>" --dry_run
-./faros_emit.sh deployment -k "<api_key>" -a "<app_name>" -e "<environmnet>" -c "<commit_sha>" --deploy_status "<deploy_status>" -p "<ci_pipeline>" --ci_org "<ci_organization>" --dry_run
-./faros_emit.sh build -k "<api_key>" -a "<app_name>" -c "<commit_sha>" --build_status "<build_status>" --ci_org "<ci_organization>" --repo "<vcs_repo>" -p "<ci_pipeline>" --vcs_source "<vcs_source>" --vcs_org "<vcs_organization>" --dry_run
+./faros_event.sh deployment -k "<api_key>" \
+    --application "<app_name>" \
+    --ci_org "<ci_organization>" \
+    --commit "<commit_sha>" \
+    --deploy_status "<deploy_status>" \
+    --environment "<environment>" \
+    --pipeline "<ci_pipeline>" \
+    --print_event
 ```
 
+### Sending a Full (Build + Deploy) Event
+
 ```sh
-# You need only replace <api_key>
-./faros_emit.sh full -k "<api_key>" -a "<app_name>" -e "Prod" -c "<commit_sha>" --build_status "Success" --deploy_status "Success" --repo "<vcs_repo>" -p "<ci_pipeline>" --ci_org "<ci_organization>" --vcs_source "<vcs_source>" --vcs_org "<vcs_organization>" --print_event
-./faros_emit.sh deployment -k "<api_key>" -a "<app_name>" -e "Prod" -c "<commit_sha>" --deploy_status "Success" -p "<ci_pipeline>" --ci_org "<ci_organization>" --print_event
-./faros_emit.sh build -k "<api_key>" -a "<app_name>" -c "<commit_sha>" --build_status "Success" --ci_org "<ci_organization>" --repo "<vcs_repo>" -p "<ci_pipeline>" --vcs_source "<vcs_source>" --vcs_org "<vcs_organization>" --print_event
+./faros_event.sh full -k "<api_key>" \
```

Comment:
why not `deployment`? since it's still a deployment event with just more info about the build, no?